op-node/rollup/derive/l1_block_info.go

```diff
@@ -259,6 +262,34 @@ func L1BlockInfoFromBytes(rollupCfg *rollup.Config, l2BlockTime uint64, data []b
 	return &info, info.unmarshalBinaryBedrock(data)
 }
 
+func (info *L1BlockInfo) marshalDepositsComplete() ([]byte, error) {
```

Comment:
Hmm i dont think that `L1BlockInfo` should be related to the `depositsComplete()` call

Reply:
This should be `marshalBinaryIsthmus` and `unmarshalBinaryIsthmus`

Reply:
should this not be for the tests of the `depositsComplete()` function?